src/helpers.js

```diff
@@ -270,12 +270,6 @@ export async function processESLintMessages(response, textEditor, showRule, work
         ret.solutions = [linterFix]
       }
     } catch (err) {
-      if (!err.message.startsWith('Line number ') &&
-        !err.message.startsWith('Column start ')
-      ) {
-        // This isn't an invalid point error from `generateRange`, re-throw it
-        throw err
-      }
```

Comment:
__*Question*__

We are guaranteed that `ruleId` will be a string or `undefined`, right?  I only ask, because if it's truthy but something other than a string, `ruleURI()` will [throw a TypeError](https://github.com/jfmengels/eslint-rule-documentation/blob/cab02691eef796e7b18ec63467220330625ccd10/index.js#L20).

Reply:
It comes straight from ESLint itself, so I'm not sure. I'll move that out of the `try` though just in case.